### PR TITLE
NOISSUE - Listing Policies

### PR DESCRIPTION
**Pull request by buraksekili**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### Which issue(s) does this PR fix/relate to?
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)

### CODE REVIEW

File: auth/keto/policies.go [hunk 1]
```diff
@@ -44,8 +46,19 @@ func (c policyAgent) CheckPolicy(ctx context.Context, pr auth.PolicyReq) error {
 	return nil
 }
 
-func (c policyAgent) AddPolicy(ctx context.Context, pr auth.PolicyReq) error {
-	trt := c.writer.TransactRelationTuples
+func (pa policyAgent) AddPolicy(ctx context.Context, pr auth.PolicyReq) error {
+	var ss *acl.Subject
+	switch isSubjectSet(pr.Subject) {
+	case true:
+		namespace, object, relation := parseSubjectSet(pr.Subject)
```
Comment (dborovcanin) [inline auth/keto/policies.go:53]: Looks like you are always checking if something is subject set before calling `parseSubjectSet` so there is probably no need to call `isSubjectSet` inside `parseSubjectSet`.

Comment (buraksekili) [reply auth/keto/policies.go:53]: I removed `isSubjectSet` inside `parseSubjectSet`

**Review by dborovcanin — DISMISSED** [review]

**Review by blokovi — DISMISSED** [review]

File: things/service.go [hunk 2]
```diff
@@ -308,21 +317,31 @@ func (ts *thingsService) ListThings(ctx context.Context, token string, pm PageMe
 		return Page{}, errors.Wrap(ErrUnauthorizedAccess, err)
 	}
 
+	subject := res.GetId()
+	if err := ts.authorize(ctx, res.GetId(), authoritiesObject, memberRelationKey); err == nil {
+		subject = "members:authorities#member"
```
Comment (mteodor) [inline things/service.go]: this should be a constant

**Review by drasko — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**